charts/refinery/values.yaml

```diff
@@ -384,3 +384,15 @@ debug:
   # Only used if debug is enabled.
   # This value will be used to set config.DebugServiceAddr if it is not already set.
   port: 6060
+
+
+deployMarker:
+  enabled: false
+  image:
+    repository: honeycombio/honeymarker
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "latest"
+  honeycombAPI: ""
+  honeycombDataset: ""
```

Comment:
The mnost future proof approach is to require via a `{{- fail }} in NOTES.txt that if `.Values.deployMarker.enabled` then `.Values.deployMarker.honeycombAPI` and `.Values.deployMarker.honeycombDataset` are set. Then in the future if we want to provide a default value from `.Values.config` or something we wont be breaking any existing users.